pom.xml

```diff
@@ -368,14 +359,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
```

Comment:
can't see why this would be needed, if suppressing doclint is still needed it can be restored